pages/help/glossary.mdx

```diff
@@ -0,0 +1,91 @@
+## General 
+
+| Acronym/Abbreviation         | Definition                                                                               |
+| ---------------------------- | ---------------------------------------------------------------------------------------- |
+| ACL                          | Access Control List                                                                      |
+| API                          | Application Programming Interface                                                        |
+| ARPA                         |  Accountable, Responsible, Participant, Advisor                                          |
+| BAM                          | Bidirectional Associative Memory                                                         |
+| BCR                          | Biospecimen Core Resource                                                                |
+| CI/CD                        |                                                                                          |
```

Comment:
I will define this once it's moved as suggested above. This might be touched upon in the "Data Processing" SOP.